code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -346,6 +346,9 @@
 		new /obj/item/weapon/gun/projectile/revolver/detective(src)
 		new /obj/item/taperoll/police(src)
 		new /obj/item/clothing/accessory/holster/armpit(src)
+		new /obj/item/clothing/glasses/sunglasses/noir(src)
+		new /obj/item/clothing/glasses/sunglasses/noir(src)
+		new /obj/item/device/flashlight/seclite(src)

```

Comment:
another reason so if some fuck goes into cryo with them on the next detective isn't left out